lib/simplecov/defaults.rb

```diff
@@ -4,8 +4,9 @@
 
 SimpleCov.profiles.define "root_filter" do
   # Exclude all files outside of simplecov root
-  root_filter = /\A#{Regexp.escape(SimpleCov.root)}/io
+  root_filter = nil
   add_filter do |src|
+    root_filter ||= /\A#{Regexp.escape(SimpleCov.root)}/io
     !(src.filename =~ root_filter)
```

Comment:
It’s unrelated to this PR, but this could be changed into `src.filename !~ root_filter`.